this can never work because musl doesn't support it; all it does is print warnings
on startup after downloading the plugin
--- a/src/core/CMakeLists.txt
+++ b/src/core/CMakeLists.txt
@@ -333,7 +333,7 @@
             dawn_use_swiftshader=false
             enable_vr=false
             enable_web_speech=false
-            enable_widevine=true
+            enable_widevine=false
             enable_library_cdms=true
             fatal_linker_warnings=false
             has_native_accessibility=false
